packages/vertexai/src/requests/request.ts

```diff
@@ -143,6 +143,15 @@ export async function makeRequest(
       body,
       requestOptions
     );
+    // Timeout is 180s by default
+    const timeoutMillis =
+      requestOptions?.timeout !== undefined
```

Comment:
I think we lost the >= 0 condition here?

Reply:
Done